packages/itwinui-css/backstop/tests/table.html

```diff
@@ -751,22 +751,22 @@ <h2>Default</h2>
       <div class="iui-table iui-condensed">
```

Comment:
Since this is a v1.0 PR can we replace the modifiers with data attributes?
```suggestion
      <div class="iui-table" data-iui-size="condensed">
```

Reply:
`iui-condensed/extra-condensed` has been replaced with `data-iui-size='condensed/extra-condensed'`